bin/main.rb

```diff
@@ -1,59 +1,73 @@
 #!/usr/bin/env ruby
-class Game
-  attr_accessor :board
+require './lib/board'
+require './lib/game'
```

Comment:
We should use `require_relative` instead of Require. Require should be used for external files, like gems, while require_relative should be used for referring to files within your directory.